src/conversions/osstr.rs

```diff
@@ -58,7 +58,22 @@ impl ToPyObject for OsStr {
 
 impl FromPyObject<'_> for OsString {
```

Comment:
I feel like this new code should probably go in `src/converstions/path.rs` rather than also adding a conversion for `OsString` ?

Reply:
Done.